src/site/stages/build/drupal/getAllPages.graphql.js

```diff
@@ -0,0 +1,31 @@
+const { gql } = require('apollo-boost');
+
+module.exports = gql`
+  {
+    nodeQuery {
+      count
+      entities {
+        ... on NodePage {
+          nid
+          entityUrl {
+            path
+            routed
+          }
+          entityBundle
+          entityPublished
+          title
+          fieldIntroText
+          fieldContentBlock {
+            entity {
+              ... on Paragraph {
+                id
+                entityBundle
+                entityRendered
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
```

Comment:
I'm definitely not an expert at GraphQL. These queries can probably be improved quite a bit.